cmd/space/generate.go

```diff
@@ -0,0 +1,65 @@
+package space
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/spf13/cobra"
+	"github.com/storacha/go-ucanto/principal/ed25519/signer"
+	"github.com/storacha/guppy/internal/cmdutil"
+)
+
+var generateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate a new space",
+	Long: wordwrap.WrapString(
+		"Generates a new Storacha space, and stores it in the local store.",
+		80),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		space, err := signer.Generate()
+		if err != nil {
+			cmd.SilenceUsage = false
+			return fmt.Errorf("generating signer for space: %w", err)
+		}
+
+		c := cmdutil.MustGetClient(*StorePathP)
+
+		err = c.AddSpace(space)
```

Comment:
Should we check we have an account to register it with before we store the space in the client? I would basically do all of create, provision and delegation before adding it to the local store. Otherwise you end up with an unusable space...